web/src/engine/websites/ComicFans.ts

```diff
@@ -0,0 +1,94 @@
+import { Tags } from '../Tags';
+import icon from './ComicFans.webp';
+import { Chapter, DecoratableMangaScraper, Manga, Page, type MangaPlugin } from '../providers/MangaPlugin';
+import * as Common from './decorators/Common';
+import { FetchJSON } from '../platform/FetchProvider';
+
+type APIPagedResult<T> = {
+    data: {
+        list: T
+    }
+}
+
+type APIResult<T> = {
+    data: T
+}
+
+type APIManga = {
+    id: number,
+    title: string
+}
+
+type APIChapter = {
+    id: number,
+    title: string,
+    chapterOrder: number
```

Comment:
Can this order number really be used in the title or is it just for internal sorting?

Reply:
Without it there is no chapter number in titles, in other words its a mess.